cmd/lifecycle/builder.go

```diff
@@ -48,39 +58,96 @@ func (b *buildCmd) Args(nargs int, args []string) error {
 }
 
 func (b *buildCmd) Privileges() error {
-	// builder should never be run with privileges
-	if priv.IsPrivileged() {
+	_, stackGroup, _, err := b.readData()
+	if err != nil {
+		return err
+	}
+
+	// builder should never be run with privileges if there aren't any stack buildpacks
+	if len(stackGroup.Group) == 0 && priv.IsPrivileged() {
```

Comment:
Does this mean the caller must know if the contents of the file specified at `stackGroupPath`? It seems like this should check for the existence of the flag instead of the contents of the file.

Reply:
Yea, I think I agree, but I was also worried about the case where:
1. the flag is not provided
2. the default file (`stack-group.toml`) is used
3. the default file is empty